input/fsh/research-subject-profile.fsh

```diff
@@ -0,0 +1,14 @@
+Extension: ResearchSubjectWithConsentStudyId
```

Comment:
the purpose of this extension is also unclear. I think the consent may record a reference to the research study but I don't understand why it's mixed with the research subject here.